### ISSUE TITLE
Enchancement request for Tables support

### ISSUE THREAD

**Issue by johnblommers** [iaacornus/simtex#11]
Consider adding Markdown tables to the list of future enhancements. Nothing fancy, just the basic table:

```markdown
Heading   | Heading    | Heading
--------- | :-------:  | ---------:
Cell      | Cell       | Cell
A         | 333        | Ditto
```

Heading | Heading | Heading
--------- | :-------:     | ---------:
Cell   | Cell  | Cell
A     | 333   | Ditto

**Comment by iaacornus** [comment]
yes it is planned to be included, there are some initial work on it, but I'm still looking on how to efficiently parse tables.

**Comment by iaacornus** [comment]
commit fd32e7aee39b3b5329cea4a95b94be2df16c34fa introduced a beta for tables, the output of the program is not as good as expected yet, although it gives a practical result, which can be viewed in [./examples/7/out/table.pdf](https://github.com/iaacornus/simtex/blob/devel/examples/7/out/table.pdf) and its input [./examples/7/table.md](https://github.com/iaacornus/simtex/blob/devel/examples/7/table.md), and another example is [./examples/8/out/table_ex.pdf](https://github.com/iaacornus/simtex/blob/devel/examples/8/out/table_ex.pdf) and its input [./examples/8/table_ex.md](https://github.com/iaacornus/simtex/blob/devel/examples/8/table_ex.md).

although there are some works to be done for full and improved support especially the syntax and the align.

- [ ] improve syntax support
- [ ] include alignment support
- [ ] improve column border (vertical line) placement
- [ ] improve row border (horizontal line) placement
